src/components/NcAppNavigationSettingsButton/NcAppNavigationSettingsButton.vue

```diff
@@ -0,0 +1,55 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<docs>
+The component to be used as a settings button in the `<NcAppNavigation>` to open `<NcAppSettingsDialog>`.
+</docs>
+
+<template>
+	<div class="app-navigation-settings-button">
+		<NcButton class="app-navigation-settings-button__button"
+			type="tertiary"
+			alignment="start"
+			wide
+			@click.stop="emit('click', $event)">
+			<template #icon>
+				<IconCog :size="20" />
+			</template>
+			<span class="app-navigation-settings-button__text">
+				{{ name }}
+			</span>
+		</NcButton>
+	</div>
+</template>
+
+<script setup>
+import { t } from '../../l10n.js'
+import NcButton from '../NcButton/index.js'
+import IconCog from 'vue-material-design-icons/Cog.vue'
+
+defineProps({
+	/**
+	 * The name of the settings button. For example, "Files Settings" or "Talk Settings".
+	 */
+	name: {
+		type: String,
+		required: false,
+		default: t('Settings'),
+	},
+})
+
+const emit = defineEmits(['click'])
+</script>
+
+<style lang="scss" scoped>
+.app-navigation-settings-button {
+	padding: var(--default-grid-baseline);
+
+	&__text {
+		// Make the settings button less presentative, like app navigation items
+		font-weight: normal;
```

Comment:
This feels every different to all other elements and is different to all current implementations, I would like to ask @nextcloud-libraries/designers first

Reply:
> This feels every different to all other elements and is different to all current implementations

I actually agree, and that's exactly what we were discussing yesterday.

But from yesterday's design review call:
- It should not be bold or secondary, otherwise it is too presentive, more than other items in the menu.
- It should be the same in all navigation menus, no matter if `NcAppNavigationItem` or `NcListItem` is used
- It should be aligned with navigation items (`NcAppNavigationItem`)

cc @jancborchardt

Reply:
To confirm – what @ShGKme listed is what I said the requirements are. The design in this PR fulfills all those requirements and looks most consistent of the possible solutions.